xblockutils/helpers.py

```diff
@@ -0,0 +1,17 @@
+"""
+Useful helper methods
+"""
+
+
+def child_isinstance(block, child_id, block_class_or_mixin):
```

Comment:
child_isinstance() is an oddly specific helper method. it's not worth holding up this PR over, but i wonder if it's asking to be something more generic. it reminds me a little of the usage_key_filter param we have in get_children() in edx-platform:

https://github.com/edx/edx-platform/blob/7338cc1286783256ee3d6b7382c9460c87d1f062/common/lib/xmodule/xmodule/x_module.py#L406